gensrc/thrift/Data.thrift

```diff
@@ -92,5 +92,9 @@ struct TStatisticData {
     9: optional string max
     10: optional string min
     11: optional string histogram
+    // global dict for low cardinality string
+    12: optional binary dict
+    // the latest tablet version for this table
+    13: optional i64 meta_version
```

Comment:
maybe it's better to use `tablet_version`？

Reply:
In fact, we get the version from the max partition VisibleVersion. Which is equal the the latest tablet version, the latest partition version, the latest table version. So we use the general conception: meta version.
```
            long version = table.getPartitions().stream().map(Partition::getVisibleVersion)
                    .max(Long::compareTo).orElse(0L);
```